src/index.scss

```diff
@@ -0,0 +1,13 @@
+
+body {
```

Comment:
drop that file and use styles in src/utils/styles/main.scss and import him to App.scss, but will be better to rename App.scss to index.scss and import index.scss in a App.js. so index.scss should have only imports without any styles